crates/cli/src/main.rs

```diff
@@ -62,31 +60,37 @@ async fn main() -> Result<()> {
             bp,
             config,
         }) => {
-            // Parse the config file here.
-            // The below is temporary to fix CI hell.
-            let _config = config;
-
-            let tokens = get_tokens();
-
-            let token0 = tokens.get(token0).unwrap();
-            let token1 = tokens.get(token1).unwrap();
-
-            let pool = Pool::new(
-                token0.clone(),
-                token1.clone(),
-                bp.parse::<u32>().unwrap(),
-                provider,
-            )
-            .await
-            .unwrap();
-
-            let pools = [pool];
-
-            for pool in pools {
-                join!(pool.monitor_pool());
+            match config {
+                true => {
+                    // If present, load config.toml and get pool from there
+                    println!("Loading config.toml...");
+                    let config_obj = config::Config::new();
+                    println!("Config.toml loaded successfully! Getting Pool...");
+                    let pool = get_pool(
+                        &config_obj.token0,
```

Comment:
the load will appear to be successful when in fact `token0` is `"unknown"`

Reply:
Good point, i replaced with a more correct statement